docs/integrations/sources/google-search-console.inapp.md

```diff
@@ -1,17 +1,25 @@
 ## Prerequisite
 
-* A verified property in Google Search Console
-* Credentials to a Google Service Account (or Google Service Account with delegated Domain Wide Authority) or Google User Account
-* Enable  Google Search Console API
-​
+- A verified property in Google Search Console
+- Google Search Console API enabled for your project
+
 ## Setup guide
-1. Enter the site URL.
-2. Enter a **Start Date** in the format YYYY-MM-DD. Data after this date will be replicated.
-2. Click **Authenticate your account** by selecting Oauth (recommended).
-   * If you prefer Service Account Key Authentication, follow the instructions in our [full documentation](https://docs.airbyte.com/integrations/sources/google-search-console/).
-3. (Optional) Set an **End Date** if you do not want data after a specific date.
-4. (Optional) Airbyte generates default reports. To add more reports, you need to add **Custom Reports** as a JSON array describing the custom reports you want to sync from Google Search Console. 
 
-Custom Reports can be added in the format: `{"name": "<report-name>", "dimensions": ["<dimension-name>", ...]}`
+1. For **Source name**, enter a name to help you identify this source.
+2. For **Website URL Property**, enter the specific website property in Google Seach Console with data you want to replicate.
+3. For **Start Date**, use the provided datepicker or enter a date in the format `YYYY-MM-DD`. Any data created on or after this date will be replicated.
+4. To authenticate the connection:
+
+   <!-- env:cloud -->
+   - **For Airbyte Cloud**: Select **Oauth** from the Authentication dropdown, then click **Sign in with Google** to authorize your account. More information on authentication methods can be found in our [full Google Search Console documentation](https://docs.airbyte.io/integrations/sources/google-search-console#setup-guide).
+   <!-- /env:cloud -->
+   <!-- env:oss -->
+   - **For Airbyte Open Source**: Select **Service Account Key Authorization** from the Authentication dropdown, then enter the **Admin Email** and **Service Account JSON Key**. For the key, copy and paste the JSON key you obtained during the service account setup. It should begin with `{"type": "service account", "project_id": YOUR_PROJECT_ID, "private_key_id": YOUR_PRIVATE_KEY, ...}`. More information on setting up a service account and generating the JSON key can be found in our [full Google Search Console documentation](https://docs.airbyte.io/integrations/sources/google-search-console#setup-guide).
```

Comment:
for reference they can still use oauth in oss but they need to create their own oauth app. IDK if we need to list out the setup process for them, but let's at least mention that

Reply:
Good call, I added the option to both docs pages and a link to Google's setup guide for creating the auth app